_data/notices.yml

```diff
@@ -48,14 +48,18 @@
 #           
 #           Sorry for the inconvenience.
            
-# - title: Restricted compute capacity on the 30.01.2021 
-#   class: info # one of [success, info, warning, danger]
-#   site: freiburg # Your site
-#   messages:
-#       - message: |
-#           Due to upgrade work to our peta-storage we need to shutdown large parts of our compute infrastructure on the
-#           30.01.2021. We are hopeful to keep the webserver running and accept jobs and workflows, but there will be a delay
-#           in processing them.
+ - title: Limited computing capacity on next April 14th and 15th, 2021
+   class: info # one of [success, info, warning, danger]
+   site: freiburg # Your site
+   messages:
+       - message: |
+           Due to upgrade work to the deNBI cloud infrastructure, we need to shut down most of our computing resources next April 14th and 15th, 2021. 
+           The shut-down will start at <b>17:00 of April 14th</b> and everything should return to normal operation by the end of the next day.
```

Comment:
is this 17 CEST? would be good to indicate I think

Reply:
Yes, and please change it if any training need more time that day

Reply:
there's just one running on that week (in the US) and I already informed them about the shutdown. I will change CET to CEST and the rest looks good to me, thanks @gmauro!

Reply:
I need some vacations :) Thank  you